kubernetes/helm/activitiinfrachart/templates/infrastructure.yml

```diff
@@ -41,6 +42,8 @@ spec:
         env:
         - name: PORT_OFFSET
           value: "22001"
+        - name: DB_VENDOR
+          value: "H2"
```

Comment:
Is that normal that you are using H2 here?

Reply:
The h2 is embedded (we've been using this all along so far) - the alternative is that we give it a database to use.